HidemichiShimura/old-portfolio
# UI update: Addition of skills used in each project

## Proposed Changes

This PR indicates the addition of the tech stacks to each project

#### Code changes

- Added constants for tech stacks of the projects
- Added tech stack icons in `<ProjectDescription />`
- Added `margin-top` to the tech stack icons for UI improvement

#### Issues affected

- Closes #73

## Additional Info

- none

## Screenshots and/or video

### Before

![image](https://user-images.githubusercontent.com/110521018/227082359-ad0f8e7e-0969-4250-b4d5-461d369fadfc.png)

### After

![image](https://user-images.githubusercontent.com/110521018/227082273-66aeecfe-0a4b-4e2b-97d1-69dfb78b7922.png)

## Testing Plan

- Check if the tech stack icons are added in each project
- Check if each project has its proper tech stack icons

## Checklist

#### Basics

- [x] Commented-out code removed
